### PR TITLE
feat(Sticky): add Suction bottom (#7531)

### PR DESCRIPTION
**Pull request by Ther-su**
### Before submitting a pull request, please make sure the following is done:

1. Read the [contributing guide](https://github.com/youzan/vant/blob/dev/.github/CONTRIBUTING.md).
2. If you've added code that should be tested, add tests.
3. If you've changed APIs, update the documentation.
4. Ensure the test suite passes (`npm test`).

#### Title Format

type(ComponentName?)：commit message

Example：

- docs: fix typo in quickstart
- build: optimize build speed
- fix(Button): incorrect style
- feat(Button): add color prop

Allowed Types:

- fix
- feat
- docs
- perf
- test
- types
- build
- chore
- refactor
- breaking change

### CODE REVIEW

**Comment by Ther-su** [comment]
I will add test case then,thougth I have met some difficulties about test now.

**Comment by Ther-su** [comment]
@chenjiahan cc.

File: src/sticky/index.wxss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.van-sticky--fixedTop {
```
Comment (chenjiahan) [inline src/sticky/index.wxss:1]: wxss 文件的作用是？

File: src/sticky/index.tsx [hunk 2]
```diff
@@ -111,14 +159,24 @@ export default createComponent({
...
+          <div ref={root} style={rootStyle}>
+            <div class={bem({ fixedTop: fixed })} style={style.value}>
+              {slots.default?.()}
+            </div>
           </div>
-        </div>
-      );
+        );
+      }
+      if (type === 'bottom') {
```
Comment (chenjiahan) [inline src/sticky/index.tsx:171]: 这里不应该把类似的 JSX 片段重复写两遍，只修改有区别的值即可

**Comment by chenjiahan** [comment]
抱歉，这个 PR 的代码质量不是很符合预期，预计不会 merge 到主分支中，建议先从简单的功能开始参与贡献喔

**Comment by Ther-su** [comment]
Thanks for your advice. I think I am not enough to make a feature PR now. And I'm sorry for the trouble I've caused you.

**Pull request closed without merging**